### PR TITLE
Events filter and categories

### PR DESCRIPTION
**Pull request by MykolasVitkus**

### CODE REVIEW

**Review by pockeviciuspovilas — APPROVED** [review]

File: .env [hunk 1]
```diff
@@ -24,12 +24,12 @@ APP_SECRET=21a982d9670576321db2088504e949a5
...
 # Configure your db driver and server_version in config/packages/doctrine.yaml
-DATABASE_URL=sqlite:///%kernel.project_dir%/var/data.db
+DATABASE_URL=changeme
 ###< doctrine/doctrine-bundle ###
 
 ###> symfony/swiftmailer-bundle ###
 # For Gmail as a transport, use: "gmail://username:password@localhost"
 # For a generic SMTP server, use: "smtp://localhost:25?encryption=&auth_mode="
 # Delivery is disabled by default via "null://localhost"
-MAILER_URL=gmail://bemantelio@gmail.com:bemanto123@localhost?encryption=tls&auth_mode=oauth
```
Comment (MartynasKasp) [inline .env]: XD

**Review by MartynasKasp — APPROVED** [review]
i love it

File: config/packages/paginator.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+knp_paginator:
+  page_range: 5                       # number of links showed in the pagination menu (e.g: you have 10 pages, a page_range of 3, on the 5th page you'll see links to page 4, 5, 6)
```
Comment (pockeviciuspovilas) [inline config/packages/paginator.yaml:2]: istrink tarpus

File: src/Controller/EventController.php [hunk 3]
```diff
@@ -2,7 +2,21 @@
```
Comment (pockeviciuspovilas) [inline src/Controller/EventController.php:2]: kodel tuscia eilute?

**Comment by MartynasKasp** [comment]
Pakeisk base branch į dev-master

**Review by DreDasLT — APPROVED** [review]

**Pull request merged**